src/main/java/nl/dtls/fairdatapoint/database/mongo/migration/production/Migration_0015_FixMetadataVersion.java

```diff
@@ -0,0 +1,123 @@
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package nl.dtls.fairdatapoint.database.mongo.migration.production;
+
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.model.Filters;
+import com.mongodb.client.model.Updates;
+import io.mongock.api.annotations.ChangeUnit;
+import io.mongock.api.annotations.Execution;
+import io.mongock.api.annotations.RollbackExecution;
+import nl.dtls.fairdatapoint.Profiles;
+import nl.dtls.fairdatapoint.entity.schema.SemVer;
+import nl.dtls.fairdatapoint.util.KnownUUIDs;
+import org.bson.Document;
+import org.springframework.context.annotation.Profile;
+import org.springframework.data.mongodb.core.MongoTemplate;
+
+@ChangeUnit(
+        id = "Migration_0015_FixMetadataVersion",
+        order = "0015",
+        author = "migrationBot"
+)
+@Profile(Profiles.PRODUCTION)
+public class Migration_0015_FixMetadataVersion {
+
+    private static final String FIELD_UUID = "uuid";
+    private static final String FIELD_VER_UUID = "versionUuid";
+    private static final String FIELD_LATEST = "latest";
+    private static final String FIELD_VERSION = "versionString";
+    private static final String FIELD_DEF = "definition";
+    private static final String COL_SCHEMAS = "metadataSchema";
+
+    private final MongoTemplate database;
+
+    private String previousVersionUuid;
+
+    public Migration_0015_FixMetadataVersion(MongoTemplate template) {
+        this.database = template;
+    }
+
+    @Execution
+    public void run() {
+        final MongoCollection<Document> schemasCol = database.getCollection(COL_SCHEMAS);
+        final Document latestResourcesSchema = schemasCol.find(
+                Filters.and(
+                        Filters.eq(FIELD_UUID, KnownUUIDs.SCHEMA_RESOURCE_UUID),
+                        Filters.eq(FIELD_LATEST, true)
+                )
+        ).first();
+        if (latestResourcesSchema == null) {
+            return;
+        }
+        previousVersionUuid = latestResourcesSchema.getString(FIELD_VER_UUID);
+        latestResourcesSchema.put(
+                FIELD_DEF,
+                latestResourcesSchema
+                        .getString(FIELD_DEF)
+                        .replace(
+                                """
+                                    sh:path dct:hasVersion ;
+                                    sh:name "version" ;
+                                    sh:nodeKind sh:Literal ;
+                                """,
+                                """
+                                    sh:path dcat:version ;
+                                    sh:nodeKind sh:Literal ;
+                                """
```

Comment:
We don't set the `sh:name` anymore?

Reply:
the name is automatically used from the `sh:path` which now will be `version` (previously would be `has version`, the explicit statement was there due to this)